### ISSUE TITLE
Tests failing because...papermill???

### ISSUE THREAD

**Issue by bradleyboehmke** [uc-python/advanced-python-datasci#23]
@eswan18, I'm getting an error in Github Actions with what appears to be a papermill problem??? Could you take a peak when you have a free moment and let me know if you can tell what the problem is?  The last notebook that runs is the 02-modeling_training.ipynb notebook which runs successfully locally.

https://github.com/uc-python/advanced-python-datasci/runs/4172955502?check_suite_focus=true#step:7:62

**Comment by eswan18** [comment]
What's the name of the kernel you're using in Jupyter? I wonder if that's the cause; it's looking for a custom-named kernel and not finding it.

We should probably both use one of the same name, which can be "uc-python" as usual or something different. And once we agree on that, I'll update the GH Action to create its kernel under than name, and I suspect that'll fix it.

**Comment by bradleyboehmke** [comment]
@eswan18 - Figured it out. Had some missing dependencies. I also removed some excessive code that I planned to replace anyways and eliminated a trial notebook.  [Tests are now passing](https://github.com/uc-python/advanced-python-datasci/actions).  

At some point in time I changed the environment name in my branch but I changed it back to "uc-python" to be consistent.  We should be good to go now.
